src/airtop/client.py

```diff
@@ -0,0 +1,90 @@
+from .base_client import BaseClient, AsyncBaseClient
+from .wrapper.windows_client import AirtopWindows, AsyncAirtopWindows
+from .wrapper.sessions_client import AirtopSessions, AsyncAirtopSessions
+
+class Airtop(BaseClient):
+    """
+    Use this class to access the different functions within the SDK. You can instantiate any number of clients with different configuration that will propagate to these functions.
+
+    Parameters
+    ----------
+    base_url : typing.Optional[str]
+        The base url to use for requests from the client.
+
+    environment : AirtopEnvironment
+        The environment to use for requests from the client. from .environment import AirtopEnvironment
+
+
+
+        Defaults to AirtopEnvironment.DEFAULT
+
+
+
+    api_key : typing.Optional[typing.Union[str, typing.Callable[[], str]]]
```

Comment:
Is api_key actually optional?

Reply:
The documentation on this specific one seemed to be outdated. I sync the doc comment from the base fern class, which doesn't show the api_key as optional.
Good catch.